src/http/ngx_http.h

```diff
@@ -77,6 +77,11 @@ typedef struct {
 } ngx_http_status_t;
 
 
+#if (NGX_HTTP_PROXY)
+#include <ngx_http_proxy_module.h>
```

Comment:
Would it be better to include it only in the specific modules that need it instead of globally?

Reply:
Done, take a look again.